File: web/src/enums/databaseType.ts
export enum DatabaseType {
  MySql = 'MySql',
  SqlServer = 'SqlServer',
  PostgreSql = 'PostgreSql',
  KingbaseES = 'KingbaseES',
  Dm = 'Dm',
}

export const DatabaseTypeOptions = [
  { label: 'MySql', value: DatabaseType.MySql },
  { label: 'SqlServer', value: DatabaseType.SqlServer },
  { label: 'PostgreSql', value: DatabaseType.PostgreSql },
  { label: '人大金仓', value: DatabaseType.KingbaseES },
  { label: '达梦', value: DatabaseType.Dm },
]

File: core/src/SSRAG/Datory/DatabaseType.cs
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using SSRAG.Datory.Annotations;

namespace SSRAG.Datory
{
    [JsonConverter(typeof(StringEnumConverter))]
    public enum DatabaseType
    {
        [DataEnum(DisplayName = "MySql")] MySql,
        [DataEnum(DisplayName = "MariaDB")] MariaDB,
        [DataEnum(DisplayName = "SqlServer")] SqlServer,
        [DataEnum(DisplayName = "PostgreSQL")] Postgres,
        [DataEnum(DisplayName = "SQLite")] SQLite,
        [DataEnum(DisplayName = "人大金仓")] Kingbase,
        [DataEnum(DisplayName = "达梦")] Dm,
        [DataEnum(DisplayName = "OceanBase")] OceanBase,
    }
}